packages/p2p/src/components/orders/orders.jsx

```diff
@@ -17,7 +17,7 @@ const Orders = observer(() => {
     order_store.setForceRerenderOrders(forceRerender);
 
     React.useEffect(() => {
-        if (general_store.active_index !== 1) general_store.setActiveIndex(1);
+        if (general_store?.active_index !== 1) general_store?.setActiveIndex(1);
```

Comment:
why do we need this change in this PR?

Reply:
If i didn't do this change the test case will fail

Reply:
It was an issue in master and is already fixed in this PR https://github.com/binary-com/deriv-app/pull/9735. Can you update your branch and check if it is working without this change?

Reply:
oriteee let me try 👌